cpp/oneapi/dal/array.hpp

```diff
@@ -24,6 +24,10 @@
 namespace oneapi::dal {
 namespace v1 {
 
+/// @tparam T The type of the memory block elements within the array.
+///              $T$ can represent any type.
```

Comment:
```suggestion
///              :literal:`T` can represent any type.
```

`$...$` is used only for math expressions, `T` here is template parameter